epubhv/epubhv.py

```diff
@@ -256,8 +269,27 @@ def convert(self):
                 old_text = element.string
                 if old_text is not None:
                     new_text = self.converter.convert(old_text)
-                    element.string.replace_with(new_text)
-            html_element.replace_with(html_element)
+                    punc = self.convert_punctuation
+                    if punc != "none":
+                        if punc == "auto":
+                            if self.convert_to is None:
+                                punc = "s2t" if method == "to_vertical" else "t2s"
+                            else:
+                                punc = self.convert_to
+                        source, target = punc.split("2")
+                        exit
```

Comment:
what is this exit?

Reply:
My L Ctrl key is broken, and I often pastes the wrong things 🥲 I haven't used pytest a lot, and it seems I sometimes tested without reinstalling the package